tests/wake-format/disallowed-input/disallowed-input.wake

```diff
@@ -21,4 +21,4 @@ def t12 = (
   # comment
 )
 
-# trailing comment
\ No newline at end of file
+# trailing comment
```

Comment:
It might be nice to keep this as no-newline to be sure the formatter adds one and only one, but this being the disallowed input, it's certainly less critical.

Reply:
ah yeah, that wasn't intentional. I think my editor made the change